api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -463,7 +463,8 @@ private void formatTimestampValues(List<DataSetPreviewValueList> valuePreviewLis
       throw new BadRequestException("Invalid Domain");
     }
 
-    return ResponseEntity.ok(dataSetService.findDataDictionaryEntry(domainValue, cdrVersion));
+    return ResponseEntity.ok(
+        dataSetService.findDataDictionaryEntry(domainValue, domain, cdrVersion));
```

Comment:
These args are confusing. What is the difference between domainValue and domain?

Reply:
domainValue is basically just value. Its from the old code, I'll change it.

Reply:
Done